src/NexusMods.App.UI/Pages/CollectionDownload/CollectionDownloadViewModel.cs

```diff
@@ -89,6 +87,24 @@ public CollectionDownloadViewModel(
             TreeDataGridAdapter.Activate();
             Disposable.Create(TreeDataGridAdapter, static adapter => adapter.Deactivate());
 
+            Observable
+                .Return(_revision)
```

Comment:
Bit of an unusual way to do it, since this will only ever return one value; I'm guessing it's done this way so `RequiredDownloadedCountObservable` can be queried Off UI in a simple way.

Fine for now. Not an issue, but it does make me curious seeing it.